com.unity.ml-agents/Runtime/Analytics/AnalyticsUtils.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using UnityEngine;
+
+namespace Unity.MLAgents.Analytics
+{
+    internal static class AnalyticsUtils
+    {
+        /// <summary>
+        /// Hash a string to remove PII or secret info before sending to analytics
+        /// </summary>
+        /// <param name="s"></param>
+        /// <returns></returns>
```

Comment:
will this cause a warning during validation?

Reply:
Will add some actual content there. It's internal, though, so probably not.